dtfs-central-api/src/services/state-machines/utilisation-report/event-handlers/helpers/get-keying-sheet-fee-payment-shares-for-fee-records.ts

```diff
@@ -105,7 +105,7 @@ const getFeePaymentsForFeeRecordPaymentEntityGroup = ({ feeRecords, payments }:
  * @returns The fee payments
  */
 export const getKeyingSheetFeePaymentSharesForFeeRecords = (matchFeeRecordsWithPayments: FeeRecordEntity[]): KeyingSheetFeePaymentShare[] => {
-  const matchFeeRecordPaymentGroups = getFeeRecordPaymentEntityGroupsFromFeeRecordEntities(matchFeeRecordsWithPayments);
+  const matchFeeRecordPaymentGroups = getFeeRecordPaymentEntityGroups(matchFeeRecordsWithPayments);
```

Comment:
See [here](https://github.com/UK-Export-Finance/dtfs2/pull/3417#discussion_r1740713202)

Reply:
Not a blocker sine this is an existing pattern, but I would expect the unit test for `getKeyingSheetFeePaymentSharesForFeeRecords`, to call any child functions it consumes. E.g:

```ts
describe('getKeyingSheetFeePaymentSharesForFeeRecords', () => {
  it('should ...', () => {
    const paymentGroups = getFeeRecordPaymentEntityGroups();  // this is the function that has been renamed

    const expected = [
      { ...paymentGroups[0], something: true },
      { ...paymentGroups[1], something: true },
    ];
  });
});
```

Currently, it seems that the unit test is testing the end result of everything, without consuming helper functions.

It's worrying that this function has ultimately changed and the test is still passing. 

Not a blocker since this seems to be an existing approach throughout.

FYI @Zainzzkk